### PR TITLE
Update sprite manifests to use new api endpoint

### PR DESCRIPTION
**Pull request by ajpal**
#32685 added the new API endpoint to separate gamelab and spritelab animation assets.
This PR updates the script we use to generate these manifests and includes the new auto-generated manifests for gamelab and spritelab.
This PR also includes changes to the defaultSprites json that we use to populate the default sprite dropdown in spritelab. Note: The changes to this file were done manually, so please actually review them :)
As part of this work I added 4 new sprites to the spritelab costume library (pig, elephant, target3, and soap). These are sprites that are already in the default dropdown, but were not available in the library itself. adding them to the library was necessary to be able to use the `/api/v1/spritelab/<version>/<category>/<animation>` endpoint.
Before:
image
After:
image

## Links

- [spec]()
- [jira]()

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/p5lab/spritelab/defaultSprites.json [hunk 1]
```diff
@@ -58,651 +58,651 @@
...
       "frameSize": {
         "x": 99,
         "y": 97
       },
       "frameCount": 1,
       "looping": true,
-      "frameDelay": 12,
-      "version": "ofSrM4hsFsE7Q_k3D7c8Ds7.JOZvkIBn"
+      "frameDelay": 2,
+      "version": "jK3q6UNbOMNbaiWBwclkhL3D8isE1cLQ.JOZvkIBn"
```
Comment (jmkulwik) [inline apps/src/p5lab/spritelab/defaultSprites.json]: Looks like this version is different than the URL

File: apps/src/p5lab/spritelab/defaultSprites.json [hunk 2]
```diff
@@ -58,651 +58,651 @@
...
-        "y": 75
+        "x": 128,
+        "y": 128
       },
       "frameCount": 1,
       "looping": true,
-      "frameDelay": 12,
-      "version": "o4EAk8JRTDT8cjYAynRaPgOufwwoxp.g"
+      "frameDelay": 2,
+      "version": "CwVno2kER.r_tECMOL4D4YL.lcaFNv7h.g"
```
Comment (jmkulwik) [inline apps/src/p5lab/spritelab/defaultSprites.json]: This version also looks different from the URL

File: apps/src/p5lab/spritelab/defaultSprites.json [hunk 3]
```diff
@@ -58,651 +58,651 @@
...
-        "y": 35
+        "x": 61,
+        "y": 37
       },
       "frameCount": 1,
       "looping": true,
-      "frameDelay": 12,
-      "version": "xRAFGMbj6fcg.qTu_vHARFRxwZbakK4l"
+      "frameDelay": 2,
+      "version": "sTV_ECoTPZ4e322cfbiihNiMroU13hpD.qTu_vHARFRxwZbakK4l"
```
Comment (jmkulwik) [inline apps/src/p5lab/spritelab/defaultSprites.json]: This one too

File: apps/src/p5lab/spritelab/defaultSprites.json (hunk 1 above)
Comment (ajpal) [reply apps/src/p5lab/spritelab/defaultSprites.json]: thanks for catching these :)

**Review by jmkulwik — APPROVED** [review]
lgtm!

**Pull request merged**